WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -59,65 +55,112 @@ public void wPScreenshotTest() {
-        waitForImagesOfTypeWithPlaceholder(R.id.image_avatar, ImageType.AVATAR);
-        waitForImagesOfTypeWithPlaceholder(R.id.image_blavatar, ImageType.BLAVATAR);
+        // Password Screen – Fill it in and click "Next"
+        // See LoginEmailPasswordFragment
+        populateTextField(R.id.input, SCREENSHOT_LOGINPASSWORD);
+        clickOn(R.id.primary_button);
+
+        // Login Confirmation Screen – Click "Continue"
+        // See LoginEpilogueFragment
+        clickOn(R.id.primary_button);
+    }
 
-        waitForRecyclerViewToStopReloading();
+    private void tmpWpLogout() {
+        // Click on the "Me" tab in the nav, then choose "Log Out"
+        clickOn(R.id.nav_me);
+        scrollToThenClickOn(R.id.row_logout);
 
-        Screengrab.screenshot("screenshot_2");
+        // Confirm that we want to log out
+        clickOn(android.R.id.button1);
     }
 
     private void editBlogPost() {
         // Choose the "sites" tab in the nav
         clickOn(R.id.nav_sites);
 
-        waitForImagesOfTypeWithPlaceholder(R.id.my_site_blavatar, ImageType.BLAVATAR);
-        Screengrab.screenshot("screenshot_3");
+        // Choose "Switch Site"
+        clickOn(R.id.switch_site);
 
+        (new SitePickerPage()).chooseSiteWithURL("infocusphotographers.com");
+
+        // Get a screenshot of the post editor
+        screenshotPostWithName("Summer Band Jam", "1-PostEditor", true);
+
+        // Get a screenshot of the drafts feature
+        screenshotPostWithName("Ideas", "5-DraftEditor", false);
+
+        // Get a screenshot of the writing feature (without image)
+        screenshotPostWithName("Time to Book Summer Sessions", "6-Writing", true);
+
+        // Exit back to the main activity
+        pressBackUntilElementIsDisplayed(R.id.nav_sites);
+    }
+
+    private void screenshotPostWithName(String name, String screenshotName, boolean hideKeyboard) {
         // Click on the "Blog Posts" row
         scrollToThenClickOn(R.id.row_blog_posts);
 
         // Wait for the blog posts to load, then edit the first post
         waitForSwipeRefreshLayoutToStopReloading();
-        waitForAtLeastOneElementWithIdToBeDisplayed(R.id.card_view);
-        clickOnChildAtIndex(0, R.id.recycler_view, R.id.card_view);
-
-        // Wait for the editor to appear and load all images
-        waitForElementToBeDisplayed(R.id.aztec);
-        waitForConditionToBeTrue(new Supplier<Boolean>() {
-            @Override
-            public Boolean get() {
-                return editPostActivityIsNoLongerLoadingImages();
-            }
-        });
+
+        PostsListPage.tapPostWithName(name);
+
+        waitForElementToBeDisplayed(R.id.editor_activity);
+
+        // Wait for the editor to load all images
+        idleFor(5000);
 
         // Click in the post title editor and ensure the caret is at the end of the title editor
-        focusEditPostTitle();
+//        focusEditPostTitle();
```

Comment:
Why is this commented out? If its not needed anymore it would be nice to remove 👍

Reply:
Good call – removed.